src/models/noteSchema.js

```diff
@@ -6,7 +6,6 @@ const noteSchema = new Schema({
     type: String,
     required: [true, "Title is required"],
     minLength: [5, "Title must be at least 5 characters"],
-    maxLength: [50, "Title must be less than 50 characters"],
```

Comment:
If the limit is 400 on the frontend, why not make it 400 on the backend?

Reply:
I wasn't sure how best to do it. To quickly change this on the front end if necessary. But if it is critical, then I can do 400 on the backend. I'm just not sure yet what value would be optimal.

Reply:
ah that makes sense! carry on!